airflow/jobs/scheduler_job.py

```diff
@@ -1275,7 +1275,7 @@ def _find_executable_task_instances(self, simple_dag_bag, session=None):
                                       " this task has been reached.", task_instance)
                         continue
 
-                if self.executor.has_task(task_instance):
+                if self.executor.is_task_running(task_instance):
```

Comment:
This change as it is will allow the scheduler to queue the same task multiple times.

Reply:
@ashb Thanks for the comment.

The base_executor.queue_command that is eventually called when this condition is `False` checks to see if the `self.queued_tasks` already has an entry for this task. Therefore, I do not see being added to this queue at this point.

Are there other points in the code you believe this could happen. Could you please point me to those points, if any?